chore(cpn): Increase number of profiles that can be created

philmoz opened the pull request:
Ran out of profile space in Companion - so increasing the number allowed.

~~TODO: Companion should notify the user if there is no space to add a new profile. Currently it fails silently.~~

pfeerick commented:
@elecpower Any reason not to do this? As I hit this limit ages ago, but left it alone as > 20 is unlikely for most users :laughing:

philmoz commented:
I've added a message box if there is no space to add a new profile.

I've also added code to move the selected profile to the top of the list when you change profile, so the most used profiles will appear higher up.

pfeerick commented:
Perfect, thank you very much! :)
